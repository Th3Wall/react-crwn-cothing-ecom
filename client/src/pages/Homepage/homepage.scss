.homepage {
    display: flex;
    flex-direction: column;
    align-items: center;
    padding: 20px 0;
}