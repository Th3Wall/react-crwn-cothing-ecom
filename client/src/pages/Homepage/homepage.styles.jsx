import styled from 'styled-components';
import { motion } from 'framer-motion';

export const HomepageContainer = styled(motion.div)`
    display: flex;
    flex-direction: column;
    align-items: center;
    padding: 20px 0;
`