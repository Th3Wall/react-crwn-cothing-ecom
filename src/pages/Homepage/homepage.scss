.homepage {
    display: flex;
    flex-direction: column;
    align-items: center;
    padding: 20px 10px;
}