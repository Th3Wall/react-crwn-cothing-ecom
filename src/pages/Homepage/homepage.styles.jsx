import styled from 'styled-components';

export const HomepageContainer = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
    padding: 20px 0;
`