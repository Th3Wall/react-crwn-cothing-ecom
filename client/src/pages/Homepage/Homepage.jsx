import Directory from '../../components/Directory/Directory';
import { HomepageContainer } from './homepage.styles';

const Homepage = () => {
    return (
        <HomepageContainer
            initial={{ opacity: 0, y: '50px' }}
            animate={{ opacity: 1, y: 0 }}
            exit={{ opacity: 0, y: '50px' }}
            transition={{ ease: "easeOut", duration: 1 }}
        >
            <Directory />
        </HomepageContainer>
    )
}

export default Homepage;