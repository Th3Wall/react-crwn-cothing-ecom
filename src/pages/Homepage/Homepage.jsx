import Directory from '../../components/Directory/Directory';
import { HomepageContainer } from './homepage.styles';

export const Homepage = () => {
    return (
        <HomepageContainer className='homepage'>
            <Directory />
        </HomepageContainer>
    )
}
