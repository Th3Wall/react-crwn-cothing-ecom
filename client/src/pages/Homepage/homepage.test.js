import { shallow } from 'enzyme';
import React from 'react';
import Homepage from './Homepage';

describe('Homepage', () => {
    it('should display homepage', () => {
        expect(shallow(<Homepage/>)).toMatchSnapshot();
    })
})