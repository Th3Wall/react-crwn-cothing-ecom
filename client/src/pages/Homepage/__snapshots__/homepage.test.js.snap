// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should display homepage 1`] = `ShallowWrapper {}`;
